src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -137,6 +142,46 @@ public function getStyle()
         return $this->style;
     }
 
+    /**
+     * Sets table style.
+     *
+     * @param int $columnIndex Column index
+     * @param TableStyle|string $name The style name or a TableStyle instance
+     *
+     * @return Table
+     */
+    public function setColumnStyle($columnIndex, $name)
+    {
+        if (!is_numeric($columnIndex)) {
```

Comment:
This should be `is_int` as 0.9 is not a valid column index but is valid for `is_numeric`